docs/lang/articles/type_system/type.md

```diff
@@ -66,26 +66,37 @@ The numeric literals in Taichi's scope have default integer or floating-point ty
 
 :::
 
-### Set default primitive type alias
+### Data type Aliases
 
-In Taichi, the keywords `int` and `float` serve as aliases for the default integer and floating-point types, respectively. These default types can be changed using the configuration option `default_ip` and `default_fp`. For instance, setting the `default_ip` to `i64` and `default_fp` to `f64` would allow you to use `int` as an alias for `i64` and `float` as an alias for `f64` in your code.
+In Taichi scope, the two names `int` and `float` serve as aliases for the default integer and floating-point types, respectively. These default types can be changed using the configuration option `default_ip` and `default_fp`. For instance, setting the `default_ip` to `i64` and `default_fp` to `f64` would allow you to use `int` as an alias for `i64` and `float` as an alias for `f64` in your code.
 
-```python
+```python skip-ci
 ti.init(default_ip=ti.i64, default_fp=ti.f64)
 
+@ti.kernel
+def example_cast() -> int:  # the returned type is ti.i64
+    x = 3.14    # x is of ti.f64 type
+    y = int(x)  # equivalent to ti.i64(x)
+```
+
+Furthermore, in the Python scope, when declaring Taichi's data containers using `ti.field`, `ti.Vector`, `ti.Matrix`, `ti.ndarray`, these two names also serve as aliases for the default integer and floating-point types. For example:
```

Comment:
`dt` of `ti.Vector` in Python scope doesn't really have effects. Let't not mention them here.